models/chainlink/ccip/chainlink_ccip_onramps.sql

```diff
@@ -0,0 +1,51 @@
+{{ config(
+        tags=['dunesql'],
+        alias = alias('ccip_onramps'),
+        post_hook='{{ expose_spells(\'["ethereum", "avalanche_c", "optimism", "polygon"]\',
+                                "project",
+                                "chainlink",
+                                \'["synthquest"]\') }}'
+        )
+}}
+
+{% set chainlink_models_on = [
+ref('chainlink_ethereum_ccip_onramps')
+, ref('chainlink_optimism_ccip_onramps')
+, ref('chainlink_avalanche_c_ccip_onramps')
+, ref('chainlink_polygon_ccip_onramps')
+
+] %}
+
+
+SELECT *
+FROM (
+    {% for onramps in chainlink_models_on %}
+    SELECT
+        origin
+        , destination
+        , source_chain_selector
+        , sequence_number
```

Comment:
This column cannot be resolved.
https://github.com/duneanalytics/spellbook/actions/runs/6406313883/job/17390662318?pr=3894#step:14:48